src/main/java/core/basesyntax/services/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.services.impl;
+
+import core.basesyntax.services.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+    @Override
+    public List<String> read(String path) {
+        try {
+            return Files.readAllLines(Path.of(path));
+        } catch (IOException e) {
+            throw new RuntimeException("Cant read from file");
```

Comment:
it's better to make it more informational
<img width="1017" alt="image" src="https://user-images.githubusercontent.com/72727656/199198625-6e57b18f-fd8f-46d8-aebf-b3f3bb0ca6ad.png">